environment.yaml

```diff
@@ -12,3 +12,9 @@ dependencies:
   - openssl
   - ripgrep
   - poetry=1.1
+  - mykrobe=0.10
+  - rasusa
+  - samtools=1.13
+  - gsl
+  - bcftools=1.13
+  - cyvcf2=0.30
```

Comment:
cyvcf2 should be a dependency in pyproject.toml, not in the conda env

Reply:
Done in https://github.com/mbhall88/tbpore/pull/8/commits/4d67fb54d1fc168f30323bca1474b8d5d56473d1